justfile

```
@@ -6,8 +6,7 @@ help:
 # setting up the pre-commit hooks.
 setup:
     poetry install
-    # setup git hooks unless environment variable HUGR_JUST_INHIBIT_GIT_HOOKS is non-empty
-    [[ -z $HUGR_JUST_INHIBIT_GIT_HOOKS ]] && poetry run pre-commit install -t pre-commit || true
+    -[[ -z "${HUGR_JUST_INHIBIT_GIT_HOOKS:-}" ]] && poetry run pre-commit install -t pre-commit
```

Is the `-` at the start of the line intentional? What does it do?

---

Okay I see - it ignores errors. Equivalent to the previous `... || true`